pyincore/analyses/pipelinedamagerepairrate/pipelinedamagerepairrate.py

```diff
@@ -200,15 +200,16 @@ def pipeline_damage_analysis_bulk_input(self, pipelines, hazard_type,
         ds_results = []
         damage_results = []
         for i, pipeline in enumerate(mapped_pipelines):
+            print("removed dfr3curve check")
```

Comment:
if this print statement is necessary, can you please change it to a log statement? Maybe a log with debug level?

Reply:
oops, thought I removed all the debug statements. Good catch